README.md

```diff
@@ -1,12 +1,60 @@
-# MeetupTalks
-
-Python Pune monthly talks
-
-# 2019
-## [January](./2019/January/README.md)
-## [February](./2019/February/README.md)
-## [March](./2019/March/README.md)
-## [April](./2019/April/README.md)
-## [May](./2019/May/README.md)
-## [June](./2019/June/README.md)
-## [July](./2019/July/README.md)
+# PythonPune Meetup Talks
+
+Welcome to meetup repository of PythonPune. This repository contains
+details of monthly events conducted as part of this group.
+
+* [Submit a talk](#submit-a-talkworkshop)
+* [Monthly newsletter](#monthly-newsletter)
+* [Summary of past events](#summary-of-past-events)
+* [Code of Conduct](#code-of-conduct)
+* [License](#license)
+
+## Submit a talk/workshop
+If you want to submit a talk for upcoming events, then you can [create
+a new
+issue](https://github.com/pythonpune/meetup-talks/issues/new/choose)
+on this repository. Just fill in the requested details and one of us
+will take care of scheduling the talk/workshop.
+
+When a talk/workshop proposal is created, it gets labeled with
+[`talk-proposal`](https://github.com/pythonpune/meetup-talks/issues?q=label:talk-proposal). Once
+it's confirmed and the date is decided,
+[`scheduled`](https://github.com/pythonpune/meetup-talks/issues?q=label:scheduled)
+label will be added to it.
+
+After the event, the event page will be updated with summary, slides
+and code snippets of the talk. Once done, the issue will be closed.
+
+## Monthly newsletter
+At the beginning of every month an issue will be created. It will have
+'Newsletter Collection Box' in it's name. Anyone can post links to
+blog posts, news articles related to Python on this issue.
+
+All the links will be compiled together and saved as
+`community_news.md` in that months directory. It will be presented
+during the meetup as well.
+
+## Summary of past events
+This is the index of summaries of past events.
+
+* 2019
+  * [January](./2019/January/README.md)
+  * [February](./2019/February/README.md)
+  * [March](./2019/March/README.md)
+  * [April](./2019/April/README.md)
+  * [May](./2019/May/README.md)
+  * [June](./2019/June/README.md)
+  * [July](./2019/July/README.md)
+
+## Code of Conduct
+Code of Conduct for [PythonPune meetup
+group](https://meetup.com/PythonPune) can be found
+[here](./CODE_OF_CONDUCT.md).
+
+## License
+This repository is licensed under [Creative Commons
+Attribution-ShareAlike 4.0 International (CC BY-SA
+4.0)](https://creativecommons.org/licenses/by-sa/4.0/). The slides,
```

Comment:
Here it is 4.0 but above https://github.com/pythonpune/meetup-talks/pull/39/files#diff-a1ee87dafebc22cbd96979f1b2b7e837R41 it is mentioned 3.0, Any reason.

Reply:
It's the license of COC. As mentioned further in the License section "The slides, code snippets or any other images in this repository might have a different license as well as copyright, please check before reusing the material."

Reply:
Ack. thanks!